examples/2d/elixir_mhd_alfven_wave.jl

```diff
@@ -36,22 +36,25 @@ analysis_callback = AnalysisCallback(semi, interval=analysis_interval, save_anal
                                      extra_analysis_integrals=(entropy, energy_total,
                                                                energy_kinetic, energy_internal,
                                                                energy_magnetic, cross_helicity))
-                                                    
+
 alive_callback = AliveCallback(analysis_interval=analysis_interval)
 
 save_solution = SaveSolutionCallback(interval=10,
                                      save_initial_solution=true,
                                      save_final_solution=true,
                                      solution_variables=:primitive)
 
-stepsize_callback = StepsizeCallback(cfl=1.0)
+cfl_number = 1.0
+stepsize_callback = StepsizeCallback(cfl=cfl_number)
+
+glm_speed_callback = GlmSpeedCallback(glm_scale=0.5, cfl_scale=cfl_number)
```

Comment:
I expect Mr. Consistency to say something about introducing three different names for the same thing :sunglasses:

Reply:
Okay, I thought it looked weird to keep writing `cfl=cfl` but I am okay with changing the names